### ISSUE TITLE
Perform reference system QA review with EVM

### ISSUE THREAD

**Issue by han-so1omon** [ingenierias-lentas/sistema#8]
## What Needs to be Done
Consider the person that would be able to complete this task and the associated materials that they would need. If no people or materials are available, create a role description so that appropriate resources may be acquired. Tasks are typically small, focused, and do not have independent deliverable functionality

*E.g. Create a list on the search index page that shows retrieved names alphabetically*

## Why it Needs to be Done
Briefly note the reason for this task. This should clearly explain why this is important, and if this can not be explained clearly, or if it has been made obsolete, then it should be simple to show that this task is no longer necessary

*E.g. Before the user has made a search request, they should see the range of possible names. Each name has an equal weight, so alphabetical ordering is a natural first ordering*

## Definition of Done
This is a formal description of the state of the task when it meets the quality measures required for the product to be delivered

This creates transparency by providing everyone with a shared understanding of what work was completed and what standards were met as part of the Task
<em>
E.g.
- [ ] All testing completed
- [ ] No known defects
- [ ] Code review completed and passed
- [ ] Documentations written and reviewed
- [ ] Meets general security requirements
- [ ] Duration
</em>

## Dependencies
Often a task depends on other tasks in order to be started or completed. The types of dependencies are:
- Finish to start: Another task must be finished before this one is able to be started
- Finish to finish: Another task must be finished before this one is able to be finished
- Start to start: Another task must be started before this one is able to be started
- Start to finish: Another task must be started before this one is able to be finished

Note the dependencies and link to them if they have associated Tasks
<em>
E.g. 
- [ ] [Task: Create generic query function with pagination](https://example.com) must be completed before this task is able to be completed
</em>

## Risks
Having a clear sense of task risks is important to prevent upcoming issues and work towards successful delivery. Common risks include:
- Scope creep
- Low performance
- High costs
- Time crunch
- Stretched resources
- Operational changes
- Lack of clarity

Brainstorm risks with the team and appropriate experts
<em>
E.g. 
- It is unclear whether the backend search function will be completed in time
- It is not clear whether users will find value in seeing the default list of user names
- This task may take longer than expected due to the developer’s experience level, and thus may cost more
</em>

## Additional Notes
Anything else that is important but has not been previously described

*E.g. Use https://example.com/ as a reference for a good example of displaying a list of names*
